### PR TITLE
Unified Config logger

### PR DESCRIPTION
**Pull request by byawitz**
## What does this PR do?

(Provide a description of what this PR does and why it's needed.)

## Test Plan

(Write your test plan here. If you changed any code, please provide us with clear instructions on how you verified your changes work. Screenshots may also be helpful.)

## Related PRs and Issues

- (Related PR or issue)

## Checklist

- [x] Have you read the [Contributing Guidelines on issues](https://github.com/appwrite/appwrite/blob/master/CONTRIBUTING.md)?
- [x] If the PR includes a change to an API's metadata (desc, label, params, etc.), does it also include updated API specs and example docs?

### CODE REVIEW

File: app/init.php [hunk 1]
```diff
@@ -724,6 +724,18 @@ function (mixed $value) {
...
     $providerConfig = System::getEnv('_APP_LOGGING_CONFIG', '');
 
+    try {
+        $loggingProvider = new DSN($providerConfig ?? '');
+
+        $providerName = $loggingProvider->getScheme();
+        $providerConfig = match ($providerName) {
+            'sentry' => ($loggingProvider->getUser() ?? '') . ';' . $loggingProvider->getHost(),
+            default => $loggingProvider->getHost(),
+        };
```
Comment (christyjacob4) [inline app/init.php:734]: https://github.com/utopia-php/logger/pull/37 

We changed the sentry & other constructors a bit to accept the parameters explicitly. We might want to adapt this PR to those changes.

File: docker-compose.yml [hunk 2]
```diff
@@ -523,8 +523,8 @@ services:
       - _APP_USAGE_STATS
       - _APP_DOCKER_HUB_USERNAME
       - _APP_DOCKER_HUB_PASSWORD
-      - _APP_LOGGING_CONFIG
```
Comment (christyjacob4) [inline docker-compose.yml:526]: We should remove all occurrences of _APP_LOGGING_PROVIDER from the compose file as well as compose.phtml

**Review by christyjacob4 — CHANGES_REQUESTED** [review]
Let's also change the target branch to 1.6.x

**Comment by stnguyen90** [comment]
Whoa there are a bit too many commits in this PR. maybe something else needs to be merged first...or some commits need to be removed 😬

**Review by stnguyen90 — COMMENTED** [review]
Whoa there are a bit too many commits in this PR. maybe something else needs to be merged first...or some commits need to be removed 😬

**Comment by byawitz** [comment]
@stnguyen90 here's a concise version of it #8087

**Pull request closed without merging**